_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -1097,12 +1097,13 @@ mfxStatus VAAPIEncoder::CreateAccelerationService(MfxVideoParam const & par)
         MFX_ERR_DEVICE_FAILED);
 
     // Configuration
-    std::vector<VAConfigAttrib> attrib(2);
+    std::vector<VAConfigAttrib> attrib(3);
 
     mfxU32 flag = VA_PROGRESSIVE;
 
     attrib[0].type = VAConfigAttribRTFormat;
     attrib[1].type = VAConfigAttribRateControl;
+    attrib[2].type = VAConfigAttribContextPriority;
```

Comment:
This resembles the following issue filed after the prev. introduction of new VAAPI feature into msdk code: #2160.  Effectively, if you introduce dependency from new VAAPI, you either need to wrap this code here in the VAAPI version check or update mediasdk libva version requirement here:
https://github.com/Intel-Media-SDK/MediaSDK/blob/81470616597c06deee3e6e1b85d6bbf5e20e0e75/builder/FindPackages.cmake#L484

I vote for the second solution to bump the version.

Reply:
Implemented the second solution.